internal/mode/static/nginx/config/http/config.go

```diff
@@ -12,12 +12,13 @@ type Server struct {
 
 // Location holds all configuration for an HTTP location.
 type Location struct {
-	Return          *Return
-	Rewrites        []string
-	Path            string
-	ProxyPass       string
-	HTTPMatchVar    string
-	ProxySetHeaders []Header
+	Return              *Return
+	Rewrites            []string
+	Path                string
+	ProxyPass           string
+	HTTPMatchVar        string
+	ProxySetHeaders     []Header
+	AddHeaderDirectives []Header
```

Comment:
This PR uses the `add_header` directives to modify HTTP response headers. The variable name `AddHeaders` might be confused with `HTTPHeaderFilter.Add`. Hence, I use `AddHeaderDirectives` here.